invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/modules/dropdown.overrides

```diff
@@ -77,3 +77,17 @@
     }
   }
 }
+
+.mobile-table-cell .ui.dropdown{
+  @media all and (max-width: @largestMobileScreen) {
+    min-width: 0em !important;
+
+    & > .menu {
+      min-width: 14em !important;
+      left: auto !important;
+      right: 0px !important;
+      border-top-width: 1px !important;
+      border-radius: 0.28571429rem 0rem 0.28571429rem 0.28571429rem;
```

Comment:
please dont hardcode values. There is a variable specifying border-radius. How to find the variable: https://inveniordm.docs.cern.ch/customize/styling-theme/